test/pylib/manager_client.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""Manager client.
+   Communicates with Manager server via socket.
+   Provides helper methods to test cases.
+   Manages driver refresh when cluster is cycled.
+"""
+
+from typing import List
+import aiohttp                                                             # type: ignore
+import aiohttp.web                                                         # type: ignore
+
+
+class ManagerClient():
+    """Helper Manager API client
+    Args:
+        sock_path (str): path to an AF_UNIX socket where Manager server is listening
+    """
+    conn: aiohttp.UnixConnector
+    session: aiohttp.ClientSession
+
+    def __init__(self, sock_path: str) -> None:
+        self.sock_path = sock_path
+
+    async def start(self):
```

Comment:
Does this have to be `async`?
If not, could it be merged with `__init__`?

(I feel like I asked this question before)

Reply:
Yes, `aiohttp.ClientSession()` needs to an `event_loop` in spite of not being a coroutine.